docs/reference/query.rst

```diff
@@ -188,6 +188,33 @@ Find all items with a file modification time between 2008-12-01 and
 
     $ beet ls 'mtime:2008-12-01..2008-12-02'
 
+You can also add an optional time value to date queries, specifying hours,
+minutes, and seconds.
+
+Times are separated from dates by a space, an uppercase 'T' or a lowercase
+'t', for example: ``2008-12-01T23:59:59``. If you specify a time, then the
+date must contain a year, month, and day. The minutes and seconds are
+optional.
+
+Here is an example that finds all items added on 2008-12-01 at or after 22:00
+but before 23:00::
+
+    $ beet ls 'added:2008-12-01T22'
```

Comment:
Sorry, confusion from my part with queries with the `..` at the end.
Cool you added the case in the test though !